bigbluebutton-html5/imports/ui/components/chat/component.jsx

```diff
@@ -31,7 +36,16 @@ export default class Chat extends Component {
         <header className={styles.header}>
           <Link className={styles.closeChat} to="/users">
             <Icon iconName="left-arrow" /> {title}
+            <Button className={styles.closeBtn}
```

Comment:
It doesn't have anything to do with the hiding of the Cam, Anton told me to add it in the same PR initially.